src/vm/database/clarity_db.rs

```diff
@@ -101,11 +101,34 @@ pub trait BurnStateDB {
     fn get_pox_prepare_length(&self) -> u32;
     fn get_pox_reward_cycle_length(&self) -> u32;
     fn get_pox_rejection_fraction(&self) -> u64;
+
+    /// Returns the header hash of the burnchain block at burnchain height `height`
+    /// using the sortition `sortition_id`.
+    ///
+    /// Returns None if `height` is negative or greater than the canonical burn
+    /// chain height.
+    ///
+    /// Panics if there is an error with the underlying database connection or
```

Comment:
Do you want to add a markdown section for "Panics"?

Reply:
Oh yeah.. if that's our convention I should.. you know it's going to add a lot of vertical height to the comments if we do this. The docs also suggest a special section for "#Errors":
https://rust-lang.github.io/api-guidelines/documentation.html

Reply:
Ya, I think that makes sense to me, since that what the convention would be with rustdoc.

Reply:
ok.. can start doing this